### DISCUSSION TITLE
TTLockOnPremise.aar in titanium module

### DISCUSSION THREAD

**Discussion by AvniShri** [appcelerator/titanium_mobile#14057] [Q&A]
Hello there!

We are trying to use an AAR file in version 12.3 but Gradle is removing this dependency during build process please suggest how to include it.

Thanks!

**Comment by m1ga** [comment]
I did already in Slack :wink:

**Comment by m1ga** [comment]
in case someone else finds this: https://www.fromzerotoapp.com/android-modules-using-local-aar-files/
I recommended to use the maven repo so it can be used in the build.gradle so there is no aar file in the app during build at all. But there is also a way for the local aar.
